manifests/params.pp

```diff
@@ -38,14 +40,25 @@
       $mcodirs = [$mco_dir]
 
       $path_separator = ':'
+
+      $user  = 0
+      $group = 0
     }
     'windows' : {
+      $package_name = 'puppet-agent'
+      $service_names = ['puppet', 'mcollective']
+
+      $local_packages_dir = 'C:\\Program Files\\Puppet Labs\\Puppet Enterprise\\packages'
```

Comment:
This may not be the correct directory in situations where the user has installed puppet elsewhere, or when running 32-bit puppet on 64-bit windows. I think there's an `env_windows_installdir` fact that you can use: https://github.com/puppetlabs/puppet_for_the_win/blob/master/conf/windows/stage/bin/environment.bat#L8 /cc @Iristyle 

You should be able to use the install_dir fact